src/app/components/find-journey/find-journey.component.ts

```diff
@@ -8,15 +14,79 @@ import {Component, OnInit} from '@angular/core';
   templateUrl: './find-journey.component.html',
   styleUrls: ['./find-journey.component.scss']
 })
-export class FindJourneyComponent implements OnInit {
+export class FindJourneyComponent {
 
-  constructor() { }
+  journey: Journey;
+  journeysFound: Journey[];
+  destination = new FormControl('');
+  id = new FormControl('');
 
-  ngOnInit() {
+  destination$: Observable<any[]>;
+  loading$: Observable<boolean>;
+  errors$: Observable<any>;
+
+  constructor(private apollo: Apollo) {
+    this.journey = new Journey();
+    this.journeysFound = [];
   }
 
   onSubmit() {
-    console.log('test');
+    console.log('id searched: ', this.id.value);
+    console.log('destination searched: ', this.destination.value);
+
+    this.searchById();
```

Comment:
I forgot but now we can do a search by destination and/or price so I purpose you to do the research with that data.

Reply:
that's my next step